bqetl_project.yaml

```diff
@@ -352,8 +352,14 @@ generate:
       - org_mozilla_ios_tiktok_reporter_tiktok_reportershare
       - org_mozilla_tiktokreporter
     events_monitoring:
-      default_event_table: events_v1
-      event_table:
-        mozillavpn: main_v1
-        accounts_frontend: accounts_events_v1
-        accounts_backend: accounts_events_v1
+      skip_pings:
```

Comment:
Should these pings be scoped per app?  Maybe with an all apps category.  `firefox_desktop_background_update_stable.background_update_v1` does exist but no events are being sent in it https://probeinfo.telemetry.mozilla.org/glean/firefox-desktop-background-update/metrics

I just looked at the table and I see there are some `nimbus_events` in the events that aren't defined in glean.  Do those need to be considered for these tables? i.e. should those be filtered out or should we also check for pings that have those events?

Reply:
Yeah, I figured to keep the configuration simple, `background-updates` can generally be ignored. The `nimbus_events` aren't super interesting, I don't think we need to explicitly filter them out, but it's unlikely that anyone will come looking for them as there are better ways to monitor nimbus.